pom.xml

```diff
@@ -112,6 +113,11 @@ TO PUBLISH SNAPSHOTS
                 <artifactId>duke-json</artifactId>
                 <version>1.3-SNAPSHOT</version>
             </dependency>
+            <dependency>
+                <groupId>no.priv.garshol.duke</groupId>
+                <artifactId>duke-es</artifactId>
```

Comment:
You should also update assembly configuration duke-dist/src/main/assembly/dep.xml to include your new module into Duke distribution